Sources/Core/Byte+Alphabet.swift

```diff
@@ -0,0 +1,178 @@
+
+// MARK: Alphabet
+
+extension Byte {
+    /// Returns whether or not the given byte is an arabic letter
+    public var isLetter: Bool {
+        return (.a ... .z).contains(self) || (.A ... .Z).contains(self)
```

Comment:
I think you meant Latin letters, not Arabic (which would be our glyphs for numbers).